src/main/java/com/vacation_bot/core/vacation/VacationService.java

```diff
@@ -0,0 +1,116 @@
+package com.vacation_bot.core.vacation;
+
+import com.vacation_bot.domain.models.UserModel;
+import com.vacation_bot.domain.models.VacationModel;
+import com.vacation_bot.domain.models.VacationTotal;
+import com.vacation_bot.repositories.UserModelRepository;
+import com.vacation_bot.repositories.VacationModelRepository;
+import com.vacation_bot.repositories.VacationTotalRepository;
+import org.springframework.stereotype.Service;
+
+import java.time.*;
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.UUID;
+
+//TODO fixes in skype
+@Service
+public class VacationService implements IVacationService {
+
+    private VacationModelRepository vacationModelRepository;
+
+    private VacationTotalRepository vacationTotalRepository;
+
+    private UserModelRepository userModelRepository;
+
+    private final int DEFAULT_VACATION_TOTAL_DAYS = 20;
```

Comment:
The constant should be created as `public static final`.

Reply:
Ok